docs/getting_started.rst

```diff
@@ -157,3 +157,11 @@ Now from your project directory, start up your containers using the following co
   ==> test02: Provisioning container "test02"...
 
 *Congrats! You're in!*
+
+Problems?
```

Comment:
I think it could be a better solution to put this into a "Troubleshouting" section that would appear in the left menu. IMO we should try to keep the "Getting Started" section as short as possible. Watcha think?